.github/workflows/ci.yml

```diff
@@ -79,7 +89,7 @@ jobs:
 
     - name: Setting CRDs for testing
       run: |
-        kubectl apply -f https://raw.githubusercontent.com/bitnami-labs/sealed-secrets/main/helm/sealed-secrets/crds/sealedsecret-crd.yaml
+        kubectl apply -f https://raw.githubusercontent.com/bitnami-labs/sealed-secrets/main/helm/sealed-secrets/crds/bitnami.com_sealedsecrets.yaml
```

Comment:
Nit: Since this has already broken once, it may be simpler to just store it in with the other test yamls in the repository versus into another repo for it.